packages/cli/src/decorators/Licensed.ee.ts

```diff
@@ -0,0 +1,15 @@
+import type { BooleanLicenseFeature } from '@/Interfaces';
```

Comment:
does this file needs to have `.ee` infix? this code will be used irrespective of if the user has an enterprise license.

Reply:
This was based on a misreading of the EE license on my part, correcting it now.